app/assets/src/components/views/discovery/DiscoverySidebar.jsx

```diff
@@ -0,0 +1,209 @@
+import React from "react";
+import cx from "classnames";
+import {
+  uniqBy,
+  sum,
+  sumBy,
+  flatten,
+  map,
+  keyBy,
+  countBy,
+  sortBy
+} from "lodash/fp";
+import moment from "moment";
+
+import PropTypes from "~/components/utils/propTypes";
+import { getAllHostGenomes } from "~/api";
+
+import cs from "./discovery_sidebar.scss";
+
+export default class DiscoverySidebar extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      stats: {
+        samples: 0,
+        projects: 0
+        // TODO (gdingle):
+        // avg_reads_per_sample: 0,
+      },
+      metadata: {
+        host: {},
+        tissue: {},
+        created_at: {}
+        // TODO (gdingle):
+        // location: {},
+      },
+      _computed: null
+    };
+  }
+
+  async componentDidMount() {
+    // TODO (gdingle): why not include genome name in payload from getSamples?
```

Comment:
that's a good idea / should probably be done at some point

Reply:
Should I add this as a Jira ticket or somewhere else? Specific label?

Reply:
sure. Could file it as a frontend task with the description